CRM/Activity/DAO/Activity.php

```diff
@@ -328,7 +328,8 @@ public static function &fields() {
           'description' => ts('FK to civicrm_option_value.id, that has to be valid, registered activity type.'),
           'required' => TRUE,
           'import' => TRUE,
-          'where' => 'civicrm_activity.activity_type_id',
+          'where' => 'civicrm_activity.activity_type_id
+          ',
```

Comment:
If a change is made to xml/schema/activity.xml, this change will get overwritten the next time GenCode is run to regenerate the DAO. I can't suggest something because I don't understand the purpose of this change - is something not adding whitespace when putting a query together and this works around that?

Reply:
@demeritcowboy this change was actually produced by GenCode. I'm not sure why...

Reply:
Ok, fixed it, the whitespace slipped in by mistake but simply running GenCode wouldn't fix it because the checksum doesn't consider whitespace.
Anyway, it's gone now.